src/a11y-dialog.ts

```diff
@@ -212,17 +212,82 @@ function moveFocusToDialog(node: HTMLElement) {
   focused.focus()
 }
 
+// Elements with these ARIA roles make their children
+// `presentational`, which nullifies their semantics.
+// @see: https://www.w3.org/TR/wai-aria/
+const PRESENTATIONAL_CHILDREN_SELECTOR = `
```

Comment:
I’m thinking it might be better to use an array that we join into a string, because a literal string like this cannot be compressed (since spaces could mean something).

Reply:
Could you expand on that?

Reply:
If we ship a string containing line breaks, the minifier will not compress it to its shortest form because it cannot know whether line breaks are relevant or not (which they are not here, since it’s only about readability). If we write this as an array instead, we preserve the readability benefits and make it optimizable:
```js
const PRESENTATIONAL_CHILDREN_SELECTOR = [
  'a',
  'b', 
  'c'
].join(',')